matrix-sdk/src/main/java/org/matrix/androidsdk/MXSession.java

```diff
@@ -2184,12 +2184,14 @@ public void enableCryptoWhenStarting() {
     /**
      * Optional set of parameters used to configure/customize the e2e encryption
      */
-    private MXCryptoConfig mCryptoConfig = null;
-    public void setCryptoConfig(@Nullable MXCryptoConfig cryptoConfig) {
+    private static MXCryptoConfig mCryptoConfig = null;
```

Comment:
Static variable should start with a `s` and it is not necessary to write `= null`. You can add `@Nullable` above this declaration